.circleci/config.yml

```diff
@@ -0,0 +1,30 @@
+defaults: &defaults
```

Comment:
I  find the commands syntax a little more readable and less verbose, though I think this is also fine.   The equivelent command would be

```
commands:
  bundle_install_and_test:
    steps:
      - checkout
      - run: bundle install
      - run: bundle exec appraisal install
      - run: bundle exec appraisal rake test
```

Then instead of `<<: *defaults`, you can use `- bundle_install_and_test`

Reply:
Since you prefer this syntax I will use it in the future

Reply:
Yeah I recommend using the 2.1 config feature from CircleCI.